documentation/docs/steps/deployIntegrationArtifact.md

```diff
@@ -0,0 +1,38 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+Deploy a CPI integration flow
+With this step you can deploy a integration flow artifact in to SAP Cloud Platform integration runtime using OData API.
+
+Learn more about the SAP Cloud Integration remote API for deploying an integration artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/08632076a1114bc1b6a1ecafef8f0178.html)
+
```

Comment:
This is not needed as it has moved to the long description in the metadata yaml file and thus will be generated here with the `${docGenDescription}` placeholder.